scripts/ci-js

```diff
@@ -3,6 +3,5 @@
 set -e
 
 npm ci
-npm test
-npm run lint
+
```

Comment:
Perhaps you could comment out the test/lint lines rather than removing them, and add a note suggesting that they are re-enabled in the future?

Reply:
Yes, agreed. You did mention that in our catchup.
Addressed in https://github.com/guardian/recipes/pull/44/commits/65080af2282462aee051b6f3eeb9edfd68989dd6